examples/persisted-store/README.md

```diff
@@ -0,0 +1,31 @@
+## About
+
+This is a basic example of store-persistence being used with `urql`.
+
+## Usage
+
+#### 1. Install dependencies in repo root.
+
+To get started with the example, make sure to install `graphcache`'s dependencies in the root directory of your clone. We `link` to the source from the `examples` directory, so its dependencies need to be installed.
+
+```bash
+# In root directory
+yarn
```

Comment:
In other examples (somewhere, I cant find them), we've done a postinstall script to save having to go up to the parent directory.

Something like
```json
"postinstall": "npm i --prefix ../../ && npm run build",
```